core/src/main/java/io/camunda/migrator/config/C8ConfigExists.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.migrator.config;
+
+import io.camunda.migrator.config.property.MigratorProperties;
+import org.springframework.boot.context.properties.bind.Binder;
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+public class C8ConfigExists implements Condition {
```

Comment:
/suggest With C8 RDBMS disabled by default, I should check if the distro still starts up without any issues since the `HistoryMigrator` bean cannot be found anymore when not configuring C8 RDBMS.